src/python/CMSSpark/data_aggregation_plots.py

```diff
@@ -0,0 +1,242 @@
+import matplotlib.pyplot as plt
+import csv
+import time
+
+
+input_file_name = '/home/justinas/Desktop/Projects/MatPlotLibPlot/all3.csv'
+
+# Open a file and read everything into dictionaries according to first line (header) of file
+# For example if header is A,B and file contains entries 1,2 and 3,4 then the output will be
+# Array with such entries: [{"A":1, "B":2"}, {"A":3, "B":"4"}]
+# This piece of code also finds "timestamp" attribute (UNIX time milliseconds),
+# converts it into YYYYMMDD date and saves as date attribute.
+rows = []
```

Comment:
In general it is bad practice to have global variables. Instead, define init function which will return rows. Then modify all other functions to take rows as input parameter.